### PR TITLE
Fixes #13770: There are still tables expectedreports & expectesreportsnodes on upgraded Rudder 4.x and 5.x,

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/13770

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 1]
```diff
@@ -491,6 +492,15 @@ upgrade_database() {
     echo " Done"
   fi
 
+  # - * => 4.3.7 : Drop expectedreports and expectesreportsnodes table 
+  RES=$(${PSQL} -t -d ${SQL_DATABASE} -c "select count(*) from information_schema.columns where lower(table_name) = 'expectedreports'")
+  if [ $RES -eq 0 ]; then
+    echo -n "INFO: Adding new 'compliance levels' table"
+    ${PSQL} -d ${SQL_DATABASE} -f ${RUDDER_UPGRADE_TOOLS}/dbMigration-4.1.x-4.1.12-add-compliancelevel-table.sql > /dev/null
```
Comment (fanf) [inline rudder-webapp/SOURCES/rudder-upgrade]: shouldn't it be "drop" ?

**Comment by ncharles** [comment]
Commit modified

File: rudder-webapp/SOURCES/rudder-upgrade (hunk 1 above)
Comment (ncharles) [reply rudder-webapp/SOURCES/rudder-upgrade]: totally

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**